functions/search.js

```diff
@@ -1,14 +1,9 @@
 import knex from 'knex';
+import { getSecret } from './utils.js';
 
 const db = knex({
   client: 'pg',
-  connection: {
-    host: 'opensgid.agrc.utah.gov',
-    user: 'agrc',
-    password: 'agrc',
-    database: 'opensgid',
-    port: 5432,
-  },
+  connection: JSON.parse(await getSecret('OPENSGID_CONNECTION_PARAMS')),
```

Comment:
**issue**: I believe the proper way to define secrets in firebase is with [these classes](https://firebase.google.com/docs/reference/functions/firebase-functions.params.md#paramsdefinesecret). Here's a plss [example](https://github.com/agrc/plss/blob/main/src/functions/storage/onCreateNotify.f.mjs)